src/panels/TcpDumpPanel.ts

```diff
@@ -24,7 +24,14 @@ const tcpDumpCommandBase = "tcpdump --snapshot-length=0 -vvv";
 const captureDir = "/tmp";
 const captureFilePrefix = "vscodenodecap_";
 const captureFileBasePath = `${captureDir}/${captureFilePrefix}`;
-const captureFilePathRegex = `${captureFileBasePath.replace(/\//g, "\\$&")}(.*)\\.cap`; // Matches the part of the filename after the prefix
+const captureFileBasePathEscaped = escapeRegExp(captureFileBasePath);
+const captureFilePathRegex = `${captureFileBasePathEscaped}(.*)\\.cap`; // Matches the part of the filename after the prefix
+
+// Escape all regex meta characters to ensure sanitation and '/' for later use in regex pattern
+function escapeRegExp(input: string): string {
```

Comment:
Thanks for this. 👍 worth to just check what is the standard `tcpDump` file name structure - I believe anything standardised file naming convention + `*.pcap` but worth to just verify please.

Reply:
Yes! you're right. ".pcap" is the standard convention. Currently ".cap" is referenced in many places so if we want to change the convention would probably be best for a separate PR. Lmn what you think.